modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectActionLocalServiceTest.java

```diff
@@ -329,6 +449,8 @@ public void testAddObjectAction() throws Exception {
 
 	@Test
 	public void testAddObjectActionWithConditionExpression() throws Exception {
+		_publishCustomObjectDefinition();
```

Comment:
Hi @guilhermedcamacho, It won't affect because it's not the same `objectDefinition`. In the `setUp` method, we are creating a new `ObjectDefinition` before each test and deleting it after each test.